manifests/plugin/discovery/params.pp

```diff
@@ -1,8 +1,8 @@
 # Default parameters for foreman::plugin::discovery
 class foreman::plugin::discovery::params {
-  $version         = 'latest'
-  $source          = 'http://downloads.theforeman.org/discovery/releases/latest/'
-  $initrd          = "foreman-discovery-image-${version}.el6.iso-img"
-  $kernel          = "foreman-discovery-image-${version}.el6.iso-vmlinuz"
+  $major_version   = 'latest'
+  $minor_version   = 'latest'
+  $source          = "http://downloads.theforeman.org/discovery/releases/${major_version}/"
+  $image_name      = "fdi-image-${minor_version}.tar"
```

Comment:
Yeah since the web server is under our control, we can always symlink/redirect in case of incompatible change on the server side.